src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.idam.IdamTokens;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingDeleteRequestPayload;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingResponse;
+
+import javax.servlet.http.HttpServletResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+
+@ExtendWith(MockitoExtension.class)
+public class HearingsServiceTest {
+
+    @Mock
+    private HmcHearingApi hmcHearingApi;
+
+    @Mock
+    private IdamService idamService;
+
+    private HearingsService hearingsService;
+    private HearingWrapper wrapper;
+    private IdamTokens idamTokens;
+    private static String CANCEL_REASON_TEMP = "AWAITING_LISTING";
+
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+
+        idamTokens = IdamTokens.builder()
+            .idamOauth2Token("token_12334132")
+            .serviceAuthorization("authorized")
+            .userId("12")
+            .build();
+
+        given(idamService.getIdamTokens()).willReturn(idamTokens);
+        hearingsService = new HearingsService(hmcHearingApi, idamService);
+    }
+
+
+
+    @DisplayName("sendDeleteHearingRequest should send request successfully")
+    @Test
+    void sendDeleteHearingRequest() {
```

Comment:
I don't think this test is doing anything other than asserting on your mocks. You set up hmcHearingApi to return expectedHearingResponse and then set the same thing as actualHearingResponse . You're comparing the same thing to itself

Reply:
Test removed